### ISSUE TITLE
Getting Content-Type: undefined and nothing is being saved

### ISSUE THREAD

**Issue by willhlaw** [adamfowleruk/generator-mljsworkplace#1]
@adamfowleruk, This looks like really good stuff and great job with the wiki. I tried to get this working with ML latest (8.0.3), but was not able to save the name of the Workspace in settings or to persist any dropped widget. I am getting a response from MarkLogic REST calls for the structure, but some other call is failing with a 400 error and Content-Type undefined. 

Here's a screenshot of the terminal output with the empty-ish workspace web page behind it.

<img width="1438" alt="mljs-workspace-screenshot" src="https://cloud.githubusercontent.com/assets/943510/8711596/6284522a-2b21-11e5-9f5a-a0b066ecf5ab.png">

Thanks in advance for any help.

**Comment by adamfowleruk** [comment]
This is likely a known issue with a lack of backwards compatibility in JSON support between V7 and V8 of MarkLogic. Are you on ML7 or 8? Which exact version?

**Comment by willhlaw** [comment]
(8.0.3)

**Comment by adamfowleruk** [comment]
Yeah this is almost certainly the same issue. It occurs because the Workplace menu bar tries to lookup menu entries in your content database. Because the search JSON structure was altered in 8.0 the REST API on the server throws a format exception. You can confirm this by looking in the JavaScript console of your browser for an error message in the response (likely XML). Please confirm just to be safe.
